src/components/Pagination/PaginationButton/PaginationButton.tsx

```diff
@@ -33,6 +33,7 @@ const PaginationButton = ({
         })}
         disabled={disabled}
         onClick={onClick}
+        type="button"
```

Comment:
This could've been done by adding `event.preventDefault()` to `onClick` handlers but this just felt like a simpler/cleaner solution.